doc/nrf/libraries/caf/power_manager.rst

```diff
@@ -25,6 +25,8 @@ By default, the timeout is set to 120 seconds.
 
 The :kconfig:`CONFIG_CAF_POWER_MANAGER_ERROR_TIMEOUT` sets the period of time after which the device is turned off upon an internal error.
 
+The :kconfig:`CONFIG_CAF_POWER_MANAGER_RUNTIME_TIMEOUT_SET` allows for the two aforementioned timeouts to be set during runtime.
```

Comment:
My suggestion for the entire section:
```
You can configure timeout with the following Kconfig options:

* :kconfig:`CONFIG_CAF_POWER_MANAGER_TIMEOUT` - This option sets the period of time after which the application enters the low power mode.
By default, the timeout is set to 120 seconds.
* :kconfig:`CONFIG_CAF_POWER_MANAGER_ERROR_TIMEOUT` - -This option sets the period of time after which the device is turned off upon an internal error.
* :kconfig:`CONFIG_CAF_POWER_MANAGER_RUNTIME_TIMEOUT_SET` - This option allows for the other two timeout configuration options to be set during run time.
```
(We don't use "aforementioned"; `run time` is written as two words in Nordic docs)